reporters_db/data/reporters.json

```diff
@@ -18376,35 +18376,6 @@
             }
         }
     ],
-    "Rodney's Notes": [
-        {
-            "cite_type": "specialty",
-            "editions": {
-                "Rodney's Notes": {
-                    "end": "1830-12-31T00:00:00",
-                    "regexes": [
-                        "$full_cite",
-                        "$volume_nominative ?$reporter, $page",
-                        "$reporter$page?"
-                    ],
-                    "start": "1792-01-01T00:00:00"
-                }
-            },
-            "examples": [
-                "Rodney's Notes, 235",
-                "1 Rodney's Notes, 235",
-                "Rodney's Notes"
-            ],
-            "mlz_jurisdiction": [
-                "us:de;"
-            ],
-            "name": "Rodney's Notes",
-            "notes": "Dates are rough, but wide, estimated from a series called Delaware Cases, which had these dates. See: http://www.worldcat.org/oclc/2882722",
-            "variations": {
```

Comment:
wait?  did we just remove Rodney's notes?

Reply:
I dont think we should remove this reporter.

Reply:
I removed this for a moment because it was causing the test to fails due missing page in the generated regex. I need to change the test: https://github.com/freelawproject/reporters-db/blob/update-reporters-cl-log/tests.py#L129 so that the page is not required. The problem here is that i don't know if this change may affect other things.

Reply:
can we put it back in ... and just drop the example without a page.